### PR TITLE
Add Implementations to repo list

### PR DESCRIPTION
**Pull request by RichardLitt**
Basically, I divided up the Repositories list to be more useful, mentioned current implementations, and added in a note about how to go about making new ones. This should close both #7 and #9.

### CODE REVIEW

**Comment by RichardLitt** [comment]
@VictorBjelkholm cc.

File: README.md [hunk 1]
```diff
@@ -23,17 +23,34 @@ In short: JSON documents with named merkle-links that can be traversed.
...
+IPLD is a multifaceted, distributed effort.
 
+### Specifications
+
+- [specs](https://github.com/ipld/specs) - The specifications for IPLD
 - [cid](https://github.com/ipld/cid) - Self-describing content-addressed identifiers for distributed systems
+
+### Implementations
+
+- [**JavaScript:** js-ipld-dag-cbor](https://github.com/ipld/js-ipld-dag-cbor) - JavaScript implementation of the IPLD spec.
```
Comment (VictorBjelkholm) [inline README.md]: js-ipld-dag-cbor is just one of the libs needed for ipld in JS if I understood @diasdavid correctly. I'm guessing we should link to both of them here.

File: README.md [hunk 2]
```diff
@@ -23,17 +23,34 @@ In short: JSON documents with named merkle-links that can be traversed.
...
+- [**Go:** ipfs/go-ipld](https://github.com/ipfs/go-ipld) - The Go implementation of IPLD.
+- [**C:** kenCode-de/c-ipld](https://github.com/kenCode-de/c-ipld) - Implementation of the IPLD spec in C.
+
+** New implementations**
```
Comment (VictorBjelkholm) [inline README.md]: Could we make this a header or attach a anchor at least so we can link to this section directly from the website?

Comment (RichardLitt) [inline README.md]: Fixed.

File: README.md (hunk 1 above)
Comment (RichardLitt) [inline README.md]: Which are the others?

Comment (diasdavid) [inline README.md]: You can find documentation about implementation of _IPLD Formats_ at the interface-ipld-format https://github.com/ipld/interface-ipld-format

Another way is by using the seach in Github org - https://github.com/ipld?utf8=%E2%9C%93&q=js-ipld-&type=&language=

Comment (RichardLitt) [inline README.md]: @diasdavid Those are in this PR already. Are there any other JS modules not in this organization that we need to think of? Have I not categorized these correctly?

Comment (RichardLitt) [inline README.md]: @diasdavid I think there is something being lost in the wires here. 

I expect an implementation to be a top level module - for instance, js-ipld. I understand than any other modules may be related to that, linked to it, dependencies, and so on. Is your understanding of module naming different than mine?

File: README.md [hunk 3]
```diff
@@ -21,19 +21,48 @@ There are a variety of systems that use merkle-tree and hash-chain inspired data
...
+
+### Specifications
 
+- [specs](https://github.com/ipld/specs) - The specifications for IPLD
 - [cid](https://github.com/ipld/cid) - Self-describing content-addressed identifiers for distributed systems
+
+### Implementations
+
+- [**JavaScript:** js-ipld-dag-cbor](https://github.com/ipld/js-ipld-dag-cbor) - JavaScript implementation of the IPLD spec.
+- [**Go:** ipfs/go-ipld](https://github.com/ipfs/go-ipld) - The Go implementation of IPLD.
```
Comment (diasdavid) [inline README.md]: Note that go-ipld isn't the IPLD implementation, in fact, it is almost out of date for 1 year

image

The implementations are:

- https://github.com/ipfs/go-ipld-btc
- https://github.com/ipfs/go-ipld-zcash
- https://github.com/ipfs/go-ipld-cbor
- https://github.com/ipfs/go-ipld-git
- https://github.com/ipfs/go-ipld-node

//cc @whyrusleeping 

@RichardLitt also read https://github.com/ipld/interface-ipld-format#background

File: README.md (hunk 1 above)
Comment (diasdavid) [inline README.md]: @RichardLitt You are missing that `IPLD` is not a spec for a single implementation, but rather a:
- data format description that can be followed by multiple implementations (dag-cbor, dag-pb, etc)
- and interface definition (interface-ipld-format)
- a tool to resolve paths  (ipld-resolver)

File: README.md [hunk 4]
```diff
@@ -21,19 +21,48 @@ There are a variety of systems that use merkle-tree and hash-chain inspired data
...
+- [**C:** kenCode-de/c-ipld](https://github.com/kenCode-de/c-ipld) - Implementation of the IPLD spec in C.
```
Comment (diasdavid) [inline README.md]: This is probably the `dag-pb` implementation only and should be updated to `c-ipld-dag-pb`, could you reachout to @kenCode-de?

Comment (RichardLitt) [inline README.md]: @kenCode-de is this accurate?

**Comment by RichardLitt** [comment]
@diasdavid How does that look?

**Comment by kenCode-de** [comment]
I have the c-ipld, c-ipfs and other repos here if you guys wanna list them: 
https://github.com/kenCode-de?tab=repositories 
 
We should be done with the first formal Release by January 2nd. I have been posting my weekly updates about the different repos here: 
https://bitsharestalk.org/index.php/topic,22576.135.html

**Comment by diasdavid** [comment]
@RichardLitt 👍

File: README.md (hunk 4 above)
Comment (kenCode-de) [inline README.md]: yep :)

Comment (kenCode-de) [inline README.md]: oh wait i thought u were referring to.. 
https://github.com/kenCode-de/c-ipld

Comment (RichardLitt) [inline README.md]: Yep, that one. Since renaming it is a larger issue, I'll just merge this here.

**Pull request merged**